tests/models/deformable_detr/test_modeling_deformable_detr.py

```diff
@@ -591,6 +591,18 @@ def create_and_check_model_fp16_forward(self):
         output = model(**inputs)["last_hidden_state"]
         self.parent.assertFalse(torch.isnan(output).any().item())
 
+    def create_and_check_model_bf16_forward(self):
```

Comment:
We need to protect this and only run if available in the environment 

```suggestion
    @require_torch_bf16
    def create_and_check_model_bf16_forward(self):
```